### PR TITLE
Update Matherion.eu

### PR DESCRIPTION
**Pull request by Fjuro**
About the web-support URL: the owner told me to add a Discord invite there

## Type of change

- [x] Update directory of a server

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/matherion/manifest.json [hunk 1]
```diff
@@ -6,11 +6,14 @@
     "%.matherion.eu"
   ],
   "social": {
-    "web": "https://matherion.eu/",
+    "web": "https://matherion.eu",
+    "web-shop": "https://matherion.craftingstore.net",
+    "web-support": "https://discord.com/invite/mXTBfuS",
```
Comment (JakobBruening) [inline minecraft_servers/matherion/manifest.json:11]: That is not the meaning behind it. [A URL to a website is to be specified here](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md). If there is no website for support, then please remove the line.
It is not a required key.

Comment (JakobBruening) [inline minecraft_servers/matherion/manifest.json:11]: That is not the meaning behind it. [A URL to a website is to be specified here](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md). If there is no website for support, then please remove the line.
It is not a required key.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/matherion/manifest.json [hunk 2]
```diff
@@ -6,11 +6,14 @@
     "%.matherion.eu"
   ],
   "social": {
-    "web": "https://matherion.eu/",
+    "web": "https://matherion.eu",
+    "web-shop": "https://matherion.craftingstore.net",
```
Comment (JakobBruening) [inline minecraft_servers/matherion/manifest.json:10]: Please use the correct key here: `web-shop` => `web_shop`
=> [Documentation reference](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#available-keys)

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by Fjuro** [comment]
Fixed, sorry

**Review by JakobBruening — APPROVED** [review]
Great! Thank you very much! ✌

**Pull request merged**